cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -0,0 +1,86 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.commands
+
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.CommandWithHelp
+import com.here.ort.model.OrtResult
+import com.here.ort.model.readValue
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
+
+import org.jetbrains.kotlin.cli.common.environment.setIdeaIoUseFallback
+
+import java.io.File
+
+import javax.script.ScriptEngineManager
+
+@Parameters(commandNames = ["evaluate"], commandDescription = "Evaluate rules on ORT result files.")
+object EvaluatorCommand : CommandWithHelp() {
+    @Parameter(description = "The ORT result file to use.",
+            names = ["--ort-result-file", "-i"],
+            required = true,
+            order = PARAMETER_ORDER_MANDATORY)
+    private lateinit var ortResultFile: File
+
+    @Parameter(description = "A rules file to use.",
+            names = ["--rules-file", "-r"],
+            order = PARAMETER_ORDER_OPTIONAL)
+    private var rulesFile: File? = null
+
+    @Parameter(description = "A rules resource to use.",
+            names = ["--rules-resource"],
+            order = PARAMETER_ORDER_OPTIONAL)
+    private var rulesResource: String? = null
+
+    private val engine = ScriptEngineManager().getEngineByExtension("kts")
+
+    override fun runCommand(jc: JCommander): Int {
+        require((rulesFile == null) != (rulesResource == null)) {
+            "Either '--rules-file' or '--rules-resource' must be specified."
+        }
+
+        // This is required to avoid
+        //
+        //     WARN: Failed to initialize native filesystem for Windows
+        //     java.lang.RuntimeException: Could not find installation home path. Please make sure bin/idea.properties
+        //     is present in the installation directory.
+        //
+        // on Windows for some reason.
+        setIdeaIoUseFallback()
+
+        val ortResult = ortResultFile.readValue<OrtResult>()
+        engine.put("ortResult", ortResult)
+
+        val preface = """
+            import com.here.ort.model.OrtResult
+            val ortResult = bindings["ortResult"] as OrtResult
+        """
+        val script = preface + (rulesFile?.readText() ?: javaClass.getResource(rulesResource).readText())
+
+        val scriptResult = engine.eval(script)
+
+        // Return 0 if the script succeeded, or 1 otherwise.
+        return if ((scriptResult as? Boolean) == true) 0 else 1
```

Comment:
What I really miss here is reporting why the result might be false. E.g. with the rule below, that GPL was found, and in which package. But I guess you plan to add this in a later PR?
My proposal would be to add the evaluator result to the ORT result file, like the analyzer and scanner do.

Reply:
I've added a new commit that requires scripts now to populate a list of errors instead of just returning a boolean.